website/config/routes.js

```diff
@@ -188,7 +179,60 @@ module.exports.routes = {
     },
   },
 
+  'GET /try-fleet/register': {
+    action: 'try-fleet/view-register',
+    locals: {
+      layout: 'layouts/layout-sandbox',
+    }
+  },
 
+  'GET /try-fleet/login': {
+    action: 'try-fleet/view-sandbox-login',
+    locals: {
+      layout: 'layouts/layout-sandbox',
+    }
+  },
+
+  'GET /try-fleet/sandbox': {
+    action: 'try-fleet/view-sandbox-or-redirect',
+    locals: {
+      layout: 'layouts/layout-sandbox',
+    },
+  },
+
+  'GET /try-fleet/sandbox-expired': {
+    action: 'try-fleet/view-sandbox-expired',
+    locals: {
+      layout: 'layouts/layout-sandbox',
+    },
+  },
+
+  'GET /try-fleet/new-sandbox': {
+    action: 'try-fleet/view-new-sandbox',
+    locals: {
+      layout: 'layouts/layout-sandbox',
+    },
+  },
+
+  'GET /try-fleet/forgot-password': {
+    action: 'entrance/view-forgot-password',
+    locals: {
+      layout: 'layouts/layout-customer',
+      pageTitleForMeta: 'Forgot password | Fleet for osquery',
+      pageDescriptionForMeta: 'Recover the password for your Fleet account.',
+      redirectToSandbox: true,
+    }
+  },
+
+  'GET /try-fleet/new-password': {
+    action: 'entrance/view-new-password',
+    locals: {
+      layout: 'layouts/layout-customer',
+      pageTitleForMeta: 'New password | Fleet for osquery',
+      pageDescriptionForMeta: 'Change the password for your Fleet account.',
+      redirectToSandbox: true,
```

Comment:
It would be more consistent to send down the `redirectToSandbox` local flag from within the view action, rather than in the routes file.  (Otherwise it's hard to know where the page script is getting it from.  And this is the consistent approach that matches how existing dynamic locals like `token` work.)

Same thing for the other places.

It's fine to leave layout, and the page title and description in the routes file.

Reply:
That makes sense, I updated the view actions and routes for these pages.